README.md

```diff
@@ -187,6 +187,22 @@ production as a docker image and push it to Docker hub at
 The image is then launched as a service in the next CI/CD job. `checklinks` is run against the
 running gatsby site in the docker container to check all links.
 
+## Process for using `devhub/latest` and `latest` tags on templates
+
+The devhub team uses the Node Template for tutorials and guides which requires making sure that both the templates and the write-ups (including solutions) are always up to date.
```

Comment:
Lisa said style for template's is not a proper noun... I think we need this in the style guide and a follow up PR to make uniform. cc @lisa-parity

Reply:
```suggestion
The devhub team uses the node template for tutorials and guides which requires making sure that both the templates and the write-ups (including solutions) are always up to date.
```